mantidproject/mantid
# Remove Typo From Multithreading.h

**Description of work.**
Simply removes a mispelling from the Interrupt macros in `Multithreading.h`. Seemed worth doing given how frequently they are used in algorithms and the like. 

**To test:**
Try a few algorithms to check they work as expected. Should be mostly covered by the unit tests. Any errors would stop the build. 


*There is no associated issue.*


*This does not require release notes* because **it is a maintenance fix with no user facing effects**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
